src/runtime/config/configuration-qemu-nvidia-gpu.toml.in

```diff
@@ -467,6 +467,14 @@ disable_selinux=@DEFDISABLESELINUX@
 # (default: true)
 disable_guest_selinux=@DEFDISABLEGUESTSELINUX@
 
+# Default number of queues for the tap device.
+#  <= 0 will be set to the same value as default_vcpus
+#  > 0 will be set to the value
+#  > defaultMaxVCPUs will be set to default_vcpus
+#  Leaving it at default should yield the best performance in a multi-stream scenario.
+#  If your application is single-stream it's better to set it at 1.
+default_tap_queues = 0
```

Comment:
I think we made an assumption later that if the  tap queues is uninitialised (in which case it is 0) use value 1 which makes sense. 
Since we are explicitly initializing it here, would prefer `default_tap_queues = 1`.
That was it aligns with the explanation as well.

Reply:
Also curious about the kind of workload you were running, where you saw better performance with single queue.

Reply:
Maybe the explanation I wrote isn't explanatory enough :)

When uninitialized (or zero) it will be set to the same value it was before the change, i.e. equal to the number of `default_vcpus`. If set at one it's going to be single-queue, which is probably not what you want in most cases.

Reply:
>  Also curious about the kind of workload you were running, where you saw better performance with single queue.

It's an Optical Proximity Correction software (calibre by Siemens)